CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to this project will be documented in this file.
 - Add support for better, more secure Keyex's and MACs with a brand new SSH backend
 - Allow manually marking domains for subdomain-level association. This will allow you to keep separate passwords for `site1.example.com` and `site2.example.com` and have them show as such in Autofill.
 - Provide better messages for OpenKeychain errors
+- Rename passwords and categories
```

Comment:
Can we actually replace categories? My understanding here is that this removes the final file if it's being moved, so we're effectively only supporting password renaming, not categories. With pass a category rename would be something like `pass mv personal/ work/` which would move all files inside, we can't do that with this PR yet. I'll change this to just 'Rename passwords' and tackle categories in a separate PR.

Reply:
I meant that you can change the category of the current password, no problem. :)